src/js/redux/actions/awardV2/awardActions.js

```diff
@@ -12,6 +12,11 @@ export const setCounts = (state) => ({
     counts: state
 });
 
+export const setAggregatedAmounts = (state) => ({
+    type: 'SET_AMOUNTS',
+    amounts: state
+});
+
```

Comment:
Could you explain why we're calling the params to most actions `state`? I think this is just a different way of thinking about it which I haven't really understood very well as of yet.

Reply:
That's a good question, and I think it's just a convention referring to the application "state" (which gets confusing for the state profile page reducer 😂). But I think you're right that it could be more clear to just call the parameter `amounts` instead of `state`, which is how they show examples in the [Redux docs](https://redux.js.org/basics/actions#action-creators)